website/content/api-docs/json-jobs.mdx

```diff
@@ -8,146 +8,234 @@ description: |-
 
 # JSON Job Specification
 
-This guide covers the JSON syntax for submitting jobs to Nomad. A useful command
-for generating valid JSON versions of HCL jobs is:
+Nomad's HTTP API uses JSON formatted job specifications except for the
+[`/job/parse` API][job-parse] which exists to convert HCL to JSON.
+
+The Nomad CLI includes a number of useful commands for working with JSON jobs.
+
+The [`nomad job run -output`][job-output] flag converts HCL jobs to JSON:
```

Comment:
This is weird enough that I had to check for myself, so maybe let's note that it doesn't actually run the job when you do that:

```suggestion
The [`nomad job run -output`][job-output] flag converts HCL jobs to JSON
without submitting the job:
```

Reply:
Great call. I still get nervous every time I run it. I was surprised to find it already prominently featured on this page of the docs!